package.json

```diff
@@ -42,11 +42,10 @@
     "@testing-library/cypress": "5.0.1",
     "@testing-library/jest-dom": "4.2.0",
     "@testing-library/react": "9.3.2",
-    "autoprefixer": "9.6.4",
-    "babel-eslint": "10.0.2",
+    "autoprefixer": "9.7.5",
     "cross-env": "6.0.3",
     "cypress": "3.4.1",
-    "eslint": "6.5.1",
+    "eslint": "6.8.0",
```

Comment:
just make it match with version `eslint` in `react-scripts` (use: npm ls eslint or yarn list eslint).